processor.py

```diff
@@ -126,7 +126,7 @@ def inverse(self):
                 cofact_r = []
                 for c in range(len(M)):
                     minor = self.get_minor(M, r, c)
-                    cofact_r.append((-1 if ((r + c) % 2) else 1) * self.det_recur(minor))
+                    cofact_r.append((-1 if (r + c) % 2 else 1) * self.det_recur(minor))
```

Comment:
Again, I like how it's clear what is being evaluated first, but if @PseudoCodeNerd likes the new one, we can approve this.

Reply:
The previous bracketing makes the calculation of the cofactor easier to understand.

Reply:
The previous bracketing makes the calculation of the cofactor easier to understand.